admin-tools/src/webapp/vm/archive/chef_archive-batch.vm

```diff
@@ -30,7 +30,7 @@
     				#end
     			</select>
     		</p>
-    		<p class="act"><input type="submit" name="eventSubmit_doBatch_Archive_PreProcess" value="$tlang.getString("archive.vm.archive1")" class="indnt1" /></p>
+    		<p class="act"><input type="submit" name="eventSubmit_doBatch_Archive_PreProcess" value="$tlang.getString("archive.vm.archive1")" class="indnt1 active" /></p>
```

Comment:
I think these are false positives. 

According to "Note 2" of https://www.w3.org/TR/WCAG20-TECHS/H44.html#H44-description 

> Note 2: The label element is not used for the following because labels for these elements are provided via the value attribute (for Submit and Reset buttons), the alt attribute (for image buttons), or element content itself (button).
>
> - Submit and Reset buttons ( input type="submit" or input type="reset")
> - Image buttons ( input type="image")
> - Hidden input fields ( input type="hidden")
> - Script buttons (button elements or <input type="button">)

I'm not sure why accesslint isn't picking up on the type nor the value in these lines.